ynr/apps/people/forms/forms.py

```diff
@@ -346,6 +347,7 @@ def save(self, commit=True, user=None):
         suggested_name = self.cleaned_data["name"]
         initial_name = self.initial["name"]
         # If the user is creating a new person, or the name has changed and doesn't match the existing name
+
         if (
```

Comment:
I think this comment goes with the code below, so I'm not sure this new white space is helpful.